tests/parser/TEST_CASES/PYTHON3.8-/conversions/edge_cases/script.yaml

```diff
@@ -0,0 +1,34 @@
+main:
+  Actor: from dff.script import Actor
+  TRANSITIONS: from dff.script import TRANSITIONS
+  MISC: from dff.script import MISC
+  some_var: from something import some_var
+  transitions: from .transitions import transitions
+  variables: import variables
+  var: some_var
+  script:
+    "'start_flow'":
+      "'start_node'":
+        TRANSITIONS:
+          transitions['two']: '(len(transitions[1]) > 0 > (- 2) > (lambda : ((- 5) + var))() > func())'
+      "'label'":
+        TRANSITIONS: transitions[7]
+        MISC:
+          '1': "{\n    2: 7,\n}[{\n    3: 2,\n}[3]]"
```

Comment:
So hard)

Reply:
This is an example to test edge cases -- it's not supposed to showcase the parser (and will not become an example).